tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,112 @@ describe('Issue model class', () => {
         });
     });
 });
+
+describe('Issue', () => {
+    let dummyTeam: Team;
+    let dummyIssue: Issue;
+    let otherDummyIssue: Issue;
+    let dummyIssueWithTeam: Issue;
+
+    let githubComment: GithubComment;
+    let newIssueDispute: IssueDispute;
+    let newTesterResponse: TesterResponse;
+
+    const noReportedDescriptionString = 'No details provided by bug reporter.\n';
+    const tutorResponseStringHeader = '# Tutor Moderation\n\n';
+
+    beforeEach(() => {
+        dummyTeam = new Team({
+            id: 'F09-2',
+            teamMembers: []
+        });
+        dummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
+        otherDummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_ASSIGNEES);
+        dummyIssueWithTeam = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+
+        githubComment = new GithubComment();
+        githubComment.body = 'Sample Text';
+
+        newIssueDispute = new IssueDispute('Cannot Work', 'Help Please');
+        newTesterResponse = new TesterResponse('Cannot Work', 'Help Please', '- [ ] Not Done', 'Reason');
+    });
+
+    it('should be initialized with the correct phase and team with clone()', () => {
+        const phaseBugReportingIssue = dummyIssue.clone(Phase.phaseBugReporting);
+        expect(phaseBugReportingIssue).toEqual(dummyIssue);
+
+        const phaseTeamResponseIssue = dummyIssueWithTeam.clone(Phase.phaseTeamResponse);
+        expect(phaseTeamResponseIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+        expect(phaseTeamResponseIssue.teamAssigned).toEqual(dummyTeam);
+
+        const phaseTesterResponseIssue = dummyIssueWithTeam.clone(Phase.phaseTesterResponse);
+        expect(phaseTesterResponseIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+
+        const phaseModerationIssue = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        expect(phaseModerationIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+        expect(phaseModerationIssue.teamAssigned).toEqual(dummyTeam);
+    });
+
+    it('should be able to get the proper Github Issue descriptions', () => {
```

Comment:
Same here, let's include the method name in the spec description:
`it('.createGithubIssueDescription() forms the correct GitHub Issue description for the issue')`